backend/dal/Migrations/v01.07.00/Down/PostDown/01-ViewProperties.sql

```diff
@@ -0,0 +1,219 @@
+﻿PRINT 'Updating View_Properties'
```

Comment:
Depending on which one of use goes first, the other will need to make sure they don't accidentally clobber the changes to this file, since the merge tool won't detect the conflict.

Reply:
You'll need to make sure the `PostDown` gets you back to the specific view before the release of `v01.06.00`

Reply:
Which should just be a copy+paste of the prior sql